.github/issue_template/bug.md

```diff
@@ -0,0 +1,56 @@
+---
+name: 🐛 Bug Report
+about: Submit a bug report to help us improve
+labels: "bug"
+---
+
+<!--
+Note: if you are reporting a wrong signature of a function or a class in
+the standard library, then the typeshed tracker is better suited
+for this report: https://github.com/python/typeshed/issues
+-->
+
+## 🐛 Bug Report
+
+(A clear and concise description of what the bug is.)
+
+### Have you read the [Contributing Guidelines](https://github.com/python/mypy/blob/master/CONTRIBUTING.md)?
+
+(Write your answer here.)
+
+## To Reproduce
+
+(Write your steps here:)
+
+1. Step 1...
+1. Step 2...
+1. Step 3...
+
+## Expected Behavior
```

Comment:
I'm not sure the "Expected behavior" and "Actual behavior" sections are useful. A lot of the time they will just be "Expected behavior: No mypy error", "Actual behavior: Got a mypy error".

Reply:
@JelleZijlstra I have added this message at end of the `bug.md` file
```
(You can freely edit this text, please remove all the lines
you believe are unnecessary.)
```